src/encodeOpReturnData.h

```diff
@@ -6,8 +6,8 @@
 #include <iomanip>
 
 // The data returned by OP_RETURN needs to be UTF-8, hex-encoded. Assuming UTF-8 for now.
-std::string encodeOpReturnData(const std::string &data) {
-    //TODO sanitize input
+inline std::string encodeOpReturnData(const std::string &data) {
+    //TODO sanitize input? what data is acceptable--just url-valid data?
     if(data.length() > 80) {
```

Comment:
Let's make OP_RETURN_LIMIT a constant.
Documentation: https://bitcoin.org/en/release/v0.12.0#relay-any-sequence-of-pushdatas-in-opreturn-outputs-now-allowed

Next question is whether to set the constant to 83 and subtract the supporting opcodes separately, or set it to 80 as measured on the data.  IDK whether we use the standard 3 bytes of supporting opcodes or not.

Again, the pendantic permissibility here is to frustrate censorship as much as possible.

Reply:
Missed replying to this earlier. I think we should set the constant to 80 since it will make more sense when checking the length of the incoming data to store. I'll get this fixed in the next PR.

As for the three bytes, I'll have to read more about it--I understand that the OP_RETURN code is one byte, followed by another byte representing the length of the data. I'm not sure where the third byte comes in.